app/gateway/cp-outage.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Control plane outage management"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+works_on:
+   - on-prem
+
+tags:
+    - control-plane
+    - data-plane
+    - resilience
+
+min_version:
+    gateway: '3.4'
+
+description: Configure data plane resilience in case of a control plane outage.
+
+related_resources:
+  - text: "{{site.konnect_short_name}} data plane nodes"
+    url: /gateway-manager/data-plane-nodes/
+  - text: "{{site.base_gateway}} control plane and data plane communication"
+    url: /gateway/cp-dp-communication/
+---
+
+{{site.base_gateway}} can be set up to support configuring new data planes in the event of a control plane outage. Data plane resilience works by designating one or more backup nodes and allowing it read/write access to a data store. This backup node will automatically push valid {{site.base_gateway}} configurations to the data store. In the event of a control plane outage, when a new node is created, it will pull the latest {{site.base_gateway}} configuration from the data store, configure itself, and start proxying requests. 
+
+This option is only recommended for users who have to adhere to strict high-availability SLAs because it requires a larger maintenance load.
+
+## How data plane resilience works
+
+When the cluster adds new data plane nodes, the control plane uses a configuration file to provision those nodes. If the control plane experiences an outage, the data plane can't be provisioned and it will silently fail until it can establish a connection with the control plane. 
```

Comment:
```suggestion
When the cluster adds new Data Plane nodes, the nodes are configured by the Control Plane using a configuration file. 

If the Control Plane experiences an outage, the Data Plane can't be provisioned and it will silently fail until it can establish a connection with the Control Plane. 
```